modules/aws/simple-host/main.tf

```diff
@@ -0,0 +1,38 @@
+terraform {
```

Comment:
I would probably rename this module to make it clear it's generated Kubernetes pods rather than other types of host.

Reply:
Any ideas? :)

alpine-with-net-attach? (seems like a bit of a mouthful)
pod-with-host-device?

Reply:
Is there anything AWS-specific about it?  Could we just put it in e.g. `modules/k8s` rather than `modules/aws`?

Reply:
Nope (in fact I was thinking of sticking all modules under just `modules/` anyway - given that's what other Terraform infra modules seem to do, and we already make quite a few AWS assumptions at the top-level of the repo).

Reply:
Hmm, if flat I'd probably be tempted by something like `linux-pod-with-net-attach`.

Bit of a mouthful but less ambiguous.